File: Aula08-Exercício/Desktop.py
from Computador import Computador


class Desktop(Computador): 
    def __init__(self,  modelo, cor, preco, potenciaDaFonte):
    
        super().__init__(modelo, cor, preco)
        self._potenciaDaFonte = potenciaDaFonte


    def cadastrar(self):
        print("Desktop cadastrado")
          


    def getInformacoes(self):
        return super().getInformacoes() +", " + self._potenciaDaFonte

File: Aula09-ExercícioEmJava/Desktop.java
public class Desktop extends Computador {
    
    protected int potenciaDaFonte;

    public Desktop(){
        super();
    }

    public Desktop(String modelo, String cor, double preco, int potencia){
        super( modelo, cor, preco );
        this.potenciaDaFonte = potencia;
    }

    @Override
    public void cadastrar(){
        System.out.println("Desktop " + this.modelo + " cadastrado");
    }

    @Override
    public String getInformacoes() {
        return super.getInformacoes() + "\nPotência da Fonte: " + this.potenciaDaFonte;
    }

    @Override
    public String toString() {
        return getInformacoes();
    }

}
